core/services/editor/BlockRegistrationManager.php

```diff
@@ -92,6 +92,26 @@ public function initialize()
         $this->initializeBlocks();
         add_action('AHEE__EE_System__initialize_last', array($this, 'registerBlocks'));
         add_action('wp_loaded', array($this, 'unloadAssets'));
+        add_filter('block_categories', array($this, 'addEspressoBlockCategories'));
+    }
+
+
+    /**
+     * @param array $categories
+     * @since $VID:$
+     * @return array
+     */
+    public function addEspressoBlockCategories(array $categories)
+    {
+        return array_merge(
+            $categories,
+            array(
+                array(
+                    'slug' => 'event-espresso',
+                    'title' => 'Event Espresso',
```

Comment:
I guess this is one of those things that shouldn't be localized because its a brand,  however we should at least wrap this in a filter because we'll need to replace this with `Event Smart` when this code eventually makes its way there.

Reply:
bah ya forgot about ES needing to change this, and it's not the first time I've done this exact same thing too :(

looks like we are just using the translation methods everywhere else for this, so I'll just do that